### PR TITLE
Allow to create PRs if a sandbox repo is used

### PR DESCRIPTION
**Pull request by betoredhat**
- At this time the Role only allows creating PRs only to the official RH certification project. This allows to use the sandbox repo if defined.

### CODE REVIEW

**Comment by betoredhat** [comment]
recheck

File: roles/create_pr/tasks/main.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@
     that: "{{ item }} is defined"
     fail_msg: "The parameter {{ item }} is required"
   with_items:
+    - project_type
```
Comment (ramperher) [inline roles/create_pr/tasks/main.yml:7]: isn't it `product_type`?

File: roles/create_pr/tasks/main.yml [hunk 2]
```diff
@@ -95,9 +96,9 @@
 
 - name: Create PR for Helm Chart Verifier
   ansible.builtin.include_tasks: helm_chart_verifier.yml
-  when: target_repository == "openshift-helm-charts/charts"
+  when: project_type == "helm"
```
Comment (ramperher) [inline roles/create_pr/tasks/main.yml:99]: isn't it `product_type`?

File: roles/create_pr/tasks/main.yml [hunk 3]
```diff
@@ -95,9 +96,9 @@
...
 
 - name: Create PR for Operator Bundle Certification
   ansible.builtin.include_tasks: bundle_operator.yml
-  when: target_repository == "redhat-openshift-ecosystem/certified-operators"
+  when: project_type == "operator"
```
Comment (ramperher) [inline roles/create_pr/tasks/main.yml:103]: isn't it `product_type`?

**Review by ramperher — COMMENTED** [review]
Some typos discovered from my side. Also, I think this PR is hitting CILAB-1375, since the app-args hint is not being processed and chart-verified is not being launched

**Comment by betoredhat** [comment]
The PR has been created on the sandbox repo https://github.com/betoredhat/charts/pull/5

**Review by ramperher — APPROVED** [review]
Thanks Beto! Looks good to me

**Pull request merged**